Gemfile

```diff
@@ -92,20 +92,20 @@ end
 gem 'kaminari'
 
 # For building and parsing XML
-gem 'nokogiri', '~> 1.6.5'
+gem 'nokogiri', '~> 1.8.1'
 gem 'equivalent-xml'
 
 # For authentication
 gem 'devise', '>= 3.4.1'
 
 # CUL Fedora Dependencies and Content Models
 gem 'cul_hydra', '~> 1.4.7'
-#gem 'cul_hydra', :path => '../cul_hydra'
+#gem 'cul_hydra', path: '../cul_hydra'
 gem 'jettywrapper', '>= 1.5.1'
 # Temporarily use specific commit because new version of gem hasn't been released yet.  Latest is 1.1.3.
-gem 'rdf-rdfxml', :github => 'ruby-rdf/rdf-rdfxml', :ref => '78c13fe5dbcecaf1f56abe9535d00f16c670a764'
+gem 'rdf-rdfxml', git: 'https://github.com/ruby-rdf/rdf-rdfxml', ref: '78c13fe5dbcecaf1f56abe9535d00f16c670a764'
```

Comment:
should be in a separate PR, but: this pin is 3 years old, and has definitely been released :)